packages/fxa-settings/src/components/GetDataTrio/index.tsx

```diff
@@ -3,12 +3,13 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React, { useCallback } from 'react';
-import { Localized, useLocalization } from '@fluent/react';
 import { copy } from '../../lib/clipboard';
 import { ReactComponent as CopyIcon } from './copy.svg';
+import { ReactComponent as NewCopyIcon } from './copy-new.svg';
```

Comment:
is this only used inline? it might be helpful to rename it something like `InlineCopyIcon` if so (just bc `NewCopyIcon` eventually becomes confusing w time)